healthtools/tests/test_scrapers.py

```diff
@@ -23,110 +26,120 @@ def setUp(self):
             self.assertEqual(response["items"][0]["index"]["_shards"]["successful"], 1)
 
     def test_nhif_inpatient_scraper_uploads_to_elasticsearch(self):
-        with open(TEST_DIR + "/dummy_files/nhif_inpatient.json", "r") as my_file:
+        self.nhif_inpatient_scraper.es_index = "healthtools-test"
+        with open(self.TEST_DATA + "dummy_files/nhif_inpatient.json", "r") as my_file:
             data = my_file.read()
-            response = self.nhif_inpatient_scraper.upload_data(json.loads(data))
+            response = self.nhif_inpatient_scraper.elasticsearch_index(json.loads(data))
             self.assertEqual(response["items"][0]["index"]["_shards"]["successful"], 1)
 
     def test_nhif_outpatient_scraper_uploads_to_elasticsearch(self):
-        with open(TEST_DIR + "/dummy_files/nhif_outpatient.json", "r") as my_file:
+        self.nhif_outpatient_scraper.es_index = "healthtools-test"
+        with open(self.TEST_DATA + "dummy_files/nhif_outpatient.json", "r") as my_file:
             data = my_file.read()
-            response = self.nhif_outpatient_scraper.upload_data(json.loads(data))
+            response = self.nhif_outpatient_scraper.elasticsearch_index(json.loads(data))
             self.assertEqual(response["items"][0]["index"]["_shards"]["successful"], 1)
 
     def test_nhif_outpatient_cs_scraper_uploads_to_elasticsearch(self):
-        with open(TEST_DIR + "/dummy_files/nhif_outpatient_cs.json", "r") as my_file:
+        self.nhif_outpatient_cs_scraper.es_index = "healthtools-test"
+        with open(self.TEST_DATA + "dummy_files/nhif_outpatient_cs.json", "r") as my_file:
             data = my_file.read()
-            response = self.nhif_outpatient_cs_scraper.upload_data(json.loads(data))
+            response = self.nhif_outpatient_cs_scraper.elasticsearch_index(json.loads(data))
             self.assertEqual(response["items"][0]["index"]["_shards"]["successful"], 1)
 
     def test_doctors_scraper_archives_to_s3(self):
-        self.doctors_scraper.s3_key = "test/doctors.json"
-        with open(TEST_DIR + "/dummy_files/doctors.json", "r") as my_file:
+        self.doctors_scraper.data_key = "test/doctors.json"
+        self.doctors_scraper.data_archive_key = "test/" + self.doctors_scraper.data_archive_key
+        with open(self.TEST_DATA + "dummy_files/doctors.json", "r") as my_file:
             data = my_file.read()
             self.doctors_scraper.archive_data(data)
         uploaded_data = self.doctors_scraper.s3.get_object(
             Bucket=AWS["s3_bucket"],
-            Key=self.doctors_scraper.s3_key
-            )["Body"].read() if AWS["s3_bucket"] else json.load(open(self.doctors_scraper.s3_key))
-        self.assertEqual(uploaded_data, data)
+            Key=self.doctors_scraper.data_key
+            )["Body"].read() if AWS["s3_bucket"] else json.load(open(self.doctors_scraper.data_key))
+        self.assertEqual(len(json.loads(uploaded_data)), len(json.loads(data)))
 
```

Comment:
This looks like it only asserts length equal instead of content equal. Let's fix this.